### PR TITLE
docs:updated quickstart

### PR DESCRIPTION
**Pull request by edlee2121**
Signed-off-by: Edward Lee <edward_lee@intuit.com>

Updated quickstart

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -53,13 +54,68 @@ kubectl logs -f simple-pipeline-out-0-xxxx main
 # Port forward the UI to https://localhost:8443/
 kubectl -n numaflow-system port-forward deployment/numaflow-server 8443:8443
 ```
-<img src="numaflow-ui-simple-pipeline.png" alt="numaflo-ui" width="650"/>
+<img src="assets/numaflow-ui-simple-pipeline.png" alt="numaflo-ui" width="650"/>
```
Comment (whynowy) [inline docs/quick-start.md:57]: Use markdown like below:

```
![Numaflow UI](assets/numaflow-ui-simple-pipeline.png)
```

**Review by whynowy — COMMENTED** [review]
1. We need the file `docs/README.md`, that is the homepage of https://numaflow.numaproj.io/;
2. Also need to make change to https://github.com/numaproj/numaflow/blob/main/mkdocs.yml.

File: docs/quick-start.md (hunk 1 above)
Comment (whynowy) [inline docs/quick-start.md:57]:
```suggestion
![Numaflow UI](assets/numaflow-ui-simple-pipeline.png)
```

Comment (edlee2121) [reply docs/quick-start.md]: Updated.

**Comment by whynowy** [comment]
@edlee2121 - You might have missed following comments.

- We need the file `docs/README.md` (Not `./README.md`. It was deleted in your previous PR), that is the homepage of https://numaflow.numaproj.io/;
- Also need to make change to https://github.com/numaproj/numaflow/blob/main/mkdocs.yml, which is used to manage the documents menu.

**Comment by edlee2121** [comment]
Thanks, restored docs/README and updated mkdocs

File: docs/quick-start.md [hunk 2]
```diff
@@ -53,13 +54,68 @@ kubectl logs -f simple-pipeline-out-0-xxxx main
...
+2022/09/07 22:29:40 (even-sink) 104
+
+# Watch the log for the odd vertex
+kubectl logs -f even-odd-odd-sink-0-a6p0n main
+2022/09/07 22:30:19 (odd-sink) 101
+2022/09/07 22:30:19 (odd-sink) 103
+```
+
+View the UI for the advanced pipleline at https://localhost:8443/
+<img src="assets/numaflow-ui-advanced-pipeline.png" alt="numaflo-ui" width="650"/>
```
Comment (whynowy) [inline docs/quick-start.md:108]:
```suggestion
![Numaflow UI](assets/numaflow-ui-advanced-pipeline.png)
```

File: docs/quick-start.md [hunk 3]
```diff
@@ -53,13 +54,68 @@ kubectl logs -f simple-pipeline-out-0-xxxx main
...
+The source code of the `even-odd` [User Defined Function](./user-defined-functions.md) can be found [here](https://github.com/numaproj/numaflow-go/tree/main/examples/function/evenodd). You also can replace the [Log](./sinks/log.md) Sink with some other sinks like [Kafka](./sinks/kafka.md) to forward the data to Kafka topics.
+
+The pipeline can be deleted by
+```shell
+kubectl delete -f https://raw.githubusercontent.com/numaproj/numaflow/stable/test/e2e/testdata/even-odd.yaml
+```
+
+## What's Next
+
+Try more examples in the [`examples`](../examples) directory.
```
Comment (whynowy) [inline docs/quick-start.md:119]:
```suggestion
Try more examples in the [`examples`](https://github.com/numaproj/numaflow/tree/main/examples) directory.
```

Should not use a relative link pointing to files/dirs out of the `docs`, when it is converted to Github pages, the link leads to a 404.

File: docs/quick-start.md (hunk 2 above)
Comment (whynowy) [reply docs/quick-start.md:108]: > 

Without using markdown, the image won't be able to display after converting to Github pages.

**Review by whynowy — APPROVED** [review]

**Pull request merged**